### PR TITLE
update kubekins experimental to bazel 0.11, bump kubekins

### PR DESCRIPTION
**Pull request by BenTheElder**
this includes the images used by test-infra
verifying everything works here: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/5137/pull-test-infra-bazel-canary/152/

so far this version has been a noop locally, bazel's release train seems to be in good shape :-)

### CODE REVIEW

**Comment by BenTheElder** [comment]
so far it's green in the test job as well, going to give it another run or two then bump things.

**Comment by BenTheElder** [comment]
test PR is thrice green with the 0.11 image, will add bump commits now.

**Comment by BenTheElder** [comment]
@krzyzacy PTAL, bump is done

File: jobs/config.json [hunk 1]
```diff
@@ -6139,7 +6139,7 @@
       "--image-family=cos-beta",
       "--image-project=cos-cloud",
       "--provider=gke",
-      "--test_args=--ginkgo.focus=\\[Feature:StackdriverMonitoring\\]|\\[Feature:StackdriverCustomMetrics\\]|\\[Feature:StackdriverMetadataAgent\\] --minStartupPods=8",
+      "--test_args=--ginkgo.focus=\\[Feature:StackdriverMonitoring\\]|\\[Feature:StackdriverCustomMetrics\\] --minStartupPods=8",
```
Comment (krzyzacy) [inline jobs/config.json]: 🤦‍♂️ someone changed generated config again, want to revert this part, or update generated configs?

we'll need a better pattern..

Comment (BenTheElder) [reply jobs/config.json]: I'll create a commit to fix it, is this the only one? :/

Comment (krzyzacy) [reply jobs/config.json]: yeah, everything else looks fine

**Comment by BenTheElder** [comment]
I think the generated jobs should be correct now.

**Comment by BenTheElder** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-canary/12287?log#log
green test run

**Pull request merged**